day_7/fizzbuzz.rb

```diff
@@ -0,0 +1,16 @@
+
+
+
+#for i in 1..100
+#updated to make availble to any number
+for i in 1..ARGV[0]
```

Comment:
In ruby, enumerables are typically preferable to for loops. How might you rewrite this so that you're utlizing the `.each enumerable`?